test/test_log_format.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+def test_log_format(docker_compose, nginxproxy):
+    r = nginxproxy.get("http://unknown.nginx-proxy.tld/port")
+    assert r.status_code == 200
+    assert r.text == "answer from port 81\n"
+    sut_container = docker_compose.containers.get("test_log_format_sut_1")
```

Comment:
The container name should match the `container_name` value in the Docker compose file:

```suggestion
    sut_container = docker_compose.containers.get("sut")
```